packages/@ember/-internals/views/lib/system/event_dispatcher.ts

```diff
@@ -98,16 +102,14 @@ export default EmberObject.extend({
     @type DOMElement
     @default 'body'
   */
-  rootElement: 'body',
+  rootElement: string | Element = 'body';
```

Comment:
`SimpleElement` maybe?

Reply:
I think we are ok with Element since events won't be dispatched in fast boot, but it's good to confirm.